styles/mixins/_organisms.scss

```diff
@@ -0,0 +1,19 @@
+@use '../variables' as var;
+
+@use 'layout';
+@use 'molecules';
+
+$-layout-space: var.$layout-space;
+
+/**
+ * An element with a frame, shadow, internal vertical rhythm and padding:
+ * serves as a container for card- and panel-like patterns.
+ */
+@mixin card {
```

Comment:
Would a `card` without hover be ever needed?

Reply:
Perhaps! I was trying to avoid prematurely over-generalizing here until we ran into that specific case.